frontend/src/components/editor-page/sidebar/specific-sidebar-entries/revisions-sidebar-entry/revision-sidebar-entry.tsx

```diff
@@ -19,13 +21,20 @@ import { Trans } from 'react-i18next'
  */
 export const RevisionSidebarEntry: React.FC<SpecificSidebarEntryProps> = ({ className, hide }) => {
   const [modalVisibility, showModal, closeModal] = useBooleanState()
+  const [revisionModalVisibility, showRevisionModal, closeRevisionModal] = useBooleanState()
 
   return (
     <Fragment>
-      <SidebarButton hide={hide} className={className} icon={IconClockHistory} onClick={showModal}>
+      <SidebarButton
+        {...cypressId('sidebar.revision.button')}
+        hide={hide}
+        className={className}
+        icon={IconClockHistory}
+        onClick={showModal}>
         <Trans i18nKey={'editor.modal.revision.title'} />
       </SidebarButton>
-      <RevisionModal show={modalVisibility} onHide={closeModal} />
+      <RevisionModal show={modalVisibility} onHide={closeModal} onShowDeleteModal={showRevisionModal} />
+      <RevisionDeleteModal show={revisionModalVisibility} onHide={closeRevisionModal} />
     </Fragment>
```

Comment:
I think it would be better to show the revision dialog again after the deletion to show the result of the process.

Reply:
done

Reply:
done